src/release/breaking-changes/page-transition-replaced-by-ZoomPageTransitionBuilder.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Page transitions replaced by ZoomPageTransitionsBuilder
+description: Using the latest page transition instead of the old one.
+---
+
+## Summary
+
+In order to keep libraries follow the latest OEM behavior,
+we've replaced the default page transition builders from
+`FadeUpwardsPageTransitionsBuilder` to
+`ZoomPageTransitionsBuilder` on all platforms excluded iOS and macOS.
+
+## Context
+
+`FadeUpwardsPageTransitionsBuilder` came with the first
+Flutter release, which defines a page transition that's
+similar to the one provided by Android O.
+
+`ZoomPageTransitionsBuilder` defines a page transition that's
+similar to the one provided by Android Q and R.
+
+According to the [Style guide for Flutter repo][], we've switched
+the default page transitions which mainly for all platforms excluded iOS and macOS.
+When the `PageTransitionsBuilder` is not found on the current platform,
```

Comment:
what does "is not found" mean?

Reply:
That is https://github.com/flutter/flutter/pull/96375/commits/d5c96054ebb763b9374a74ec7ebc5840036645a6/packages/flutter/lib/src/material/page_transitions_theme.dart#L598-599

```dart
final PageTransitionsBuilder matchingBuilder =
    builders[platform] ?? const ZoomPageTransitionsBuilder();
```

Reply:
sorry to clarify when i ask a question like that what i really mean is "please update the text to answer the following question: ..." :-)